sources/3.0.0/sections/12-metadata.adoc

```diff
@@ -604,18 +604,8 @@ S-102 uses S100_DataCoverage without modification.
 |*Mandatory in S-102* +
 A single value may be provided when all axes have a common resolution. +
 For multiple value provision, use axis order as specified in dataset. +
-May be approximate for ungeorectified data (*not applicable to this edition of S-102*). +
-For example, for 5-metre resolution, the value 5 must be encoded. +
-See note immediately below.
-
+For example, for 5-metre resolution, the value 5 must be encoded.
```

Comment:
Add an example in case of a geographic grid.

Reply:
An example of exactly what? Do you have a suggestion?
The note can be brought back, it says:

`NOTE: If the grid cell size varies over the extent of the grid, an approximated value based on model parameters or production metadata should be used.
`

or it can be amended to something like:

`NOTE: If the grid cell size varies over the extent of the grid (when using a geographic grid), an approximated value based on model parameters or production metadata should be used.
`

Reply:
Restoring the note is also a solution:

`NOTE: If the grid cell size varies over the extent of the grid (i.e., when using a geographic grid), an approximated value should be used.`

Reply:
I added the following to remarks:
`If the grid cell size varies over the extent of the grid +
(i.e., when using a geographic grid), an approximated value should be used.
`